components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -2431,17 +2433,20 @@ def archived_files(request, iid=None, conn=None, **kwargs):
         for w in wells:
             images.append(w.getWellSample(index).image())
     if len(images) == 0:
-        logger.debug(
-            "Cannot download archived file becuase Images not found.")
-        return HttpResponseServerError(
-            "Cannot download archived file because Images not found (ids:"
-            " %s)." % (imgIds))
+        message = 'Cannot download archived file because Images not ' \
+            'found (ids: %s)' % (imgIds)
+        logger.debug(message)
```

Comment:
Is this `logger.debug` intentional?

Reply:
I think @chris-allan just wanted to keep the logging the same here.

Reply:
Sorry, makes sense.